.changeset/wicked-schools-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/hydrogen': patch
+---
+
+Update cart mutation to query for cart warnings
```

Comment:
I think this should probably include an outline of the inventory warnings change, and maybe even a link to the Storefront API changelog item